src/SWP/Bundle/ContentBundle/Resources/config/serializer/CmfRoutingBundle/Model.Route.yml

```diff
@@ -0,0 +1,21 @@
+Symfony\Cmf\Bundle\RoutingBundle\Model\Route:
```

Comment:
and `Symfony\Cmf\Bundle\RoutingBundle\Doctrine\Phpcr\Route` extends `Symfony\Cmf\Bundle\RoutingBundle\Model\Route` which have his own not needed for api properties.